src/App.tsx

```diff
@@ -25,39 +24,40 @@ const App = () => {
 
         setIsLoaded(true);
         setTodos(preparedTodosList);
-        setInitialTodos(preparedTodosList);
       });
   };
 
-  const sortTitles = () => {
-    setTodos([...initialTodos]
-      .sort((a, b) => a.title.localeCompare(b.title)));
-  };
+  const sortingMethod = (sort: string) => {
+    switch (sort) {
+      case 'title':
+        return setTodos([...todos]
```

Comment:
you do the same again, don't set sorted todos in the state, just return a new array and call it after `render(...`